gpsTimeCheck.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+import numpy as np
+import ruamel.yaml
+
+from gps_time.core import GPSTime
```

Comment:
Can you add the pip insall command ? Should this be located at the top level root folder